tests/acceptance/features/apiContract/propfindShares.feature

```diff
@@ -147,3 +147,51 @@ Feature: propfind a shares
       | old              | %file_id_pattern%  |
       | new              | %file_id_pattern%  |
       | spaces           | %share_id_pattern% |
+
+  @issue-8510
+  Scenario Outline: check share-id from PROPFIND request to shared items (Personal Space)
+    Given using spaces DAV path
+    And using SharingNG
+    And user "Alice" has created folder "folderToShare"
+    And user "Alice" has uploaded file with content "some content" to "testfile.txt"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | <resource> |
+      | space           | Personal   |
+      | sharee          | Brian      |
+      | shareType       | user       |
+      | permissionsRole | Viewer     |
+    And user "Brian" has a share "<resource>" synced
+    And user "Brian" sends PROPFIND request from the space "Shares" to the resource "/" using the WebDAV API
+    Then the HTTP status code should be "207"
+    And as user "Brian" the PROPFIND response should contain a resource "<resource>" with these key and value pairs:
+      | key        | value           |
+      | oc:shareid | %last_share_id% |
+    Examples:
+      | resource      |
+      | folderToShare |
+      | testfile.txt  |
```

Comment:
I would recommend testing for different dav paths and either file or folder should be enough.